polkadot/node/core/backing/src/tests/prospective_parachains.rs

```diff
@@ -1,1742 +0,0 @@
-		let pvd = dummy_pvd();
-		let validation_code = ValidationCode(vec![1, 2, 3]);
-
-		let expected_head_data = test_state.head_data.get(&para_id).unwrap();
-
-		let pov_hash = pov.hash();
-		let candidate_a = TestCandidateBuilder {
-			para_id,
-			relay_parent: leaf_parent,
-			pov_hash,
-			head_data: expected_head_data.clone(),
-			erasure_root: make_erasure_root(&test_state, pov.clone(), pvd.clone()),
-			persisted_validation_data_hash: pvd.hash(),
-			validation_code: validation_code.0.clone(),
-		};
-		let mut candidate_b = candidate_a.clone();
-		candidate_b.relay_parent = leaf_grandparent;
-
-		let candidate_a = candidate_a.build();
-		let candidate_b = candidate_b.build();
-
-		for candidate in &[candidate_a, candidate_b] {
-			let second = CandidateBackingMessage::Second(
-				leaf_hash,
-				candidate.to_plain(),
-				pvd.clone(),
-				pov.clone(),
-			);
-
-			virtual_overseer.send(FromOrchestra::Communication { msg: second }).await;
-
-			assert_validate_seconded_candidate(
-				&mut virtual_overseer,
-				candidate.descriptor().relay_parent,
-				&candidate,
-				&pov,
-				&pvd,
-				&validation_code,
-				expected_head_data,
-				false,
-			)
-			.await;
-
-			// `seconding_sanity_check`
-			let hypothetical_candidate = HypotheticalCandidate::Complete {
-				candidate_hash: candidate.hash(),
-				receipt: Arc::new(candidate.clone()),
-				persisted_validation_data: pvd.clone(),
-			};
-			let expected_request_a = vec![(
-				HypotheticalMembershipRequest {
-					candidates: vec![hypothetical_candidate.clone()],
-					fragment_chain_relay_parent: Some(leaf_hash),
-				},
-				make_hypothetical_membership_response(hypothetical_candidate, leaf_hash),
-			)];
-			assert_hypothetical_membership_requests(
-				&mut virtual_overseer,
-				expected_request_a.clone(),
-			)
-			.await;
-
-			// Prospective parachains are notified.
-			assert_matches!(
-				virtual_overseer.recv().await,
-				AllMessages::ProspectiveParachains(
-					ProspectiveParachainsMessage::IntroduceSecondedCandidate(
-						req,
-						tx,
-					),
-				) if
-					&req.candidate_receipt == candidate
-					&& req.candidate_para == para_id
-					&& pvd == req.persisted_validation_data
-				=> {
-					tx.send(true).unwrap();
-				}
-			);
-
-			assert_matches!(
-				virtual_overseer.recv().await,
-				AllMessages::StatementDistribution(
-					StatementDistributionMessage::Share(
-						parent_hash,
-						_signed_statement,
-					)
-				) if parent_hash == candidate.descriptor().relay_parent => {}
-			);
-
-			assert_matches!(
-				virtual_overseer.recv().await,
-				AllMessages::CollatorProtocol(CollatorProtocolMessage::Seconded(hash, statement)) => {
-					assert_eq!(candidate.descriptor().relay_parent, hash);
-					assert_matches!(statement.payload(), Statement::Seconded(_));
-				}
-			);
-		}
-
-		virtual_overseer
-	});
-}
-
-// Test that the candidate reaches quorum successfully.
-#[test]
-fn backing_works() {
```

Comment:
This test is completely removed since it overlaps with `backing_works` from mod.rs